src/commands/logs/functions.ts

```diff
@@ -28,39 +30,40 @@ function getLog(logData: { level: string; message: string }) {
 }
 
 export const logsFunction = async (functionName: string | undefined, options: OptionValues, command: BaseCommand) => {
+  intro('logs:function')
   const client = command.netlify.api
   const { site } = command.netlify
   const { id: siteId } = site
 
   if (options.level && !options.level.every((level: string) => LOG_LEVELS_LIST.includes(level))) {
-    log(`Invalid log level. Choices are:${CLI_LOG_LEVEL_CHOICES_STRING}`)
+    NetlifyLog.warn(`Invalid log level. Choices are:${CLI_LOG_LEVEL_CHOICES_STRING}`)
   }
 
   const levelsToPrint = options.level || LOG_LEVELS_LIST
 
   const { functions = [] } = await client.searchSiteFunctions({ siteId })
 
   if (functions.length === 0) {
-    log(`No functions found for the site`)
-    return
+    NetlifyLog.error(`No functions found for the site`)
   }
 
   let selectedFunction
   if (functionName) {
-    selectedFunction = functions.find((fn: any) => fn.n === functionName)
+    selectedFunction = functions.find((fn: { n: string }) => fn.n === functionName)
   } else {
-    const { result } = await inquirer.prompt({
-      name: 'result',
-      type: 'list',
+    const result = await select({
```

Comment:
I've managed to narrow down the failing tests to this select prompt but I have no idea why that would cause the timeouts. Digging further...

Reply:
Ah it's because we previously mocked the interaction for inquirer but we need to do the same here for the clack prompt